src/screens/FinancialSummary/FinancialSummaries.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import BaseHeader from '../../components/BaseComponents/BaseHeader';
+import { TextField, Typography, Button } from '@material-ui/core';
+import OutlinedCardList from '../../components/OutlinedCardList';
+import { RouteComponentProps } from 'react-router-dom';
+import SearchIcon from '@material-ui/icons/Search';
+import { connect } from 'react-redux';
+import { RootState } from '../../lib/redux/store';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { FinancialSummaryRecord } from '../../lib/airtable/interface';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+      height: '100vh',
+    },
+    content: {
+      textAlign: 'left',
+      color: theme.palette.text.secondary,
+      padding: '0px 20px 80px',
+      width: '100%',
+      height: '80vh',
+      overflow: 'auto',
+    },
+    payButton: {
+      borderRadius: '12px',
+      height: '30px',
+      width: '80px',
+      color: 'white',
+      backgroundColor: theme.palette.primary.main,
+    },
+    viewButton: {
+      borderRadius: '12px',
+      height: '30px',
+      width: '80px',
+      color: theme.palette.primary.main,
+      backgroundColor: 'white',
+      border: `1px solid ${theme.palette.primary.main}`,
+    },
+    searchContainer: {
+      float: 'right',
+      padding: '0px 30px',
+    },
+    searchIcon: {
+      fontSize: '20px',
+      color: theme.palette.text.primary,
+      marginTop: '-5px',
+    },
+    search: {
+      border: `1px solid ${theme.palette.primary.main}`,
+      borderRadius: '10px',
+      width: '130px',
+      height: '25px',
+      padding: '2px 10px',
+    },
+    title: {
+      marginTop: '20px',
+    },
+  });
+
+interface FinancialSummariesProps extends RouteComponentProps {
+  classes: {
+    root: string;
+    content: string;
+    payButton: string;
+    viewButton: string;
+    searchIcon: string;
+    searchContainer: string;
+    search: string;
+    title: string;
+  };
+  financialSummaries?: any;
+  location: any;
+}
+
+function FinancialSummaries(props: FinancialSummariesProps) {
+  const { classes, financialSummaries } = props;
+  let paidFinancialSummaries: any[] = [];
+  let unpaidFinancialSummaries: any[] = [];
+
+  const formatFinancialSummaries = (summaries: FinancialSummaryRecord[]) => {
+      summaries.forEach(summary => {
+        const entry = { number: summary.totalAmountBilled, label: summary.lastUpdated, unit: 'MMK'};
+        if (summary.totalCustomersBilled === summary.totalCustomersPaid) {
+          paidFinancialSummaries.push(entry);
+        } else {
+          unpaidFinancialSummaries.push(entry);
+        }
+      });
+  };
+
+  formatFinancialSummaries(financialSummaries);
+
+  const getSearch = () => {
+    return (
+      <div className={classes.searchContainer}>
+        <TextField
+          className={classes.search}
```

Comment:
The Figma has a date range rather than a text search. 

We don't have to have functionality for the date range now, but if it's not too much trouble could the frontend portion of it be added (no functionality required yet)? If you don't have time, we can delete this search part for now and just add in a date range later in a future sprint.

![image](https://user-images.githubusercontent.com/35501399/108929548-24d24a80-75f9-11eb-91c5-c767a719ca27.png)

Reply:
I'll have to look into this more, for now I'll delete the search. Are there any Material UI samples out there for what we're looking for?